CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## v0.4 [2021-05-28]
+
+_Bug fixes_
+
+- Duplicate local variables have now been removed across all the files
```

Comment:
Duplicate or Unused?

Reply:
Duplicate, since the local variable has been repeated across the files